src/main/java/net/mcreator/ui/action/ActionRegistry.java

```diff
@@ -179,6 +176,8 @@ public class ActionRegistry {
 	public final BasicAction imageEditorRedo;
 	public final BasicAction imageEditorSave;
 	public final BasicAction imageEditorSaveAs;
+	public final ImageEditorCopyAction imageEditorCopy;

```

Comment:
What about `ImageEditorCutAction` action? It could behave like this one followed by a `Removal`.

P. S. I know this is draft for now, I just wanted to give the idea before it vanished from my own head entirely.

Reply:
Yes, this is a draft and will remain one for probably a long time. It's just here so I can keep better track of what I've already done and what I still want to do.

Reply:
Should be added now.